src/Blueprint/Blueprint/help/Get-AzBlueprint.md

```diff
@@ -0,0 +1,214 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Blueprint.dll-Help.xml
+Module Name: Az.Blueprint
+online version: https://docs.microsoft.com/en-us/powershell/module/az.blueprint/get-azblueprint
+schema: 2.0.0
+---
+
+# Get-AzBlueprint
+
+## SYNOPSIS
+Get one or more blueprint definitions.
+
+## SYNTAX
+
+### SubscriptionScope (Default)
+```
+Get-AzBlueprint [[-SubscriptionId] <String>] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### BySubscriptionAndName
+```
+Get-AzBlueprint [[-SubscriptionId] <String>] [-Name] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### BySubscriptionNameAndVersion
+```
+Get-AzBlueprint [[-SubscriptionId] <String>] [-Name] <String> [-Version] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### BySubscriptionNameAndLatestPublished
+```
+Get-AzBlueprint [[-SubscriptionId] <String>] [-Name] <String> [-LatestPublished]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ManagementGroupScope
+```
+Get-AzBlueprint [-ManagementGroupId] <String> [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByManagementGroupAndName
+```
+Get-AzBlueprint [-ManagementGroupId] <String> [-Name] <String> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+### ByManagementGroupNameAndVersion
+```
+Get-AzBlueprint [-ManagementGroupId] <String> [-Name] <String> [-Version] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByManagementGroupNameAndLatestPublished
+```
+Get-AzBlueprint [-ManagementGroupId] <String> [-Name] <String> [-LatestPublished]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get one or more blueprint definitions. Blueprint definitions exist at the management group or subscription scope.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS> Get-AzBlueprint
```

Comment:
@filizt for the examples you provide in the cmdlet help files, would you mind including the resulting output of each command so users have an idea of what is returned?